### PR TITLE
NXP-25821: add drop content message to dropzone

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: themes/light/theme.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     --nuxeo-app-font: 'Open Sans', Arial, sans-serif;
     --nuxeo-text-default: #32353f;
 
-    --nuxeo-primary-color: #00aded;
+    --nuxeo-primary-color: rgb(0, 173, 237);
```
Comment (Gabez0r) [inline themes/light/theme.html]: Why change this to rgb?

Comment (nmpcunha) [reply themes/light/theme.html]: Not intended, happened while getting the matching rgb combination.

**Comment by guirenard** [comment]
:+1:

**Comment by Gabez0r** [comment]
👍

**Pull request merged**